### DISCUSSION TITLE
Feature Request: @vanilla-extract/vite-plugin add peer dependency of latest Vite major version

### DISCUSSION THREAD

**Discussion by bestickley** [vanilla-extract-css/vanilla-extract#1052] [General]
When I install `@vanilla-extract/vite-plugin` with the latest major Vite version I get the below warning message.

```
 WARN  Issues with peer dependencies found
ui
└─┬ @vanilla-extract/vite-plugin 3.7.0
  └── ✕ unmet peer vite@"^2.2.3 || ^3.0.0": found 4.1.4
```
Could the vite peer dependency of `@vanilla-extract/vite-plugin` include ^4.0.0?

**Comment by askoufis** [comment]
[The latest version of the vite plugin](https://github.com/vanilla-extract-css/vanilla-extract/blob/ab72774f28c9bb839c97fdee6e7c11e1a82276dc/packages/vite-plugin/package.json#L27), `v3.8.0`, has an updated peer dependency range that addresses this issue.

**Reply by bestickley** [reply]
@askoufis, not sure how I missed this. Thank you!
